lib/collection/src/shards/forward_proxy_shard.rs

```diff
@@ -235,7 +247,7 @@ impl ForwardProxyShard {
     /// # Cancel safety
     ///
     /// This method is cancel safe.
-    async fn read_batch_with_hashring(
+    async fn read_batch_with_hashring_filter(
```

Comment:
Have to agree with rabbit here, name & comments are changed but not a fucntionality. Let's revert renaming and comments and keep filtering only for non-resharding case

Reply:
(or actually implement the filter)